docs/using_unidist/unidist_on_dask.rst

```diff
@@ -0,0 +1,37 @@
+..
+      Copyright (C) 2021-2022 Modin authors
+
+      SPDX-License-Identifier: Apache-2.0
+
+:orphan:
+
+unidist on Dask
```

Comment:
Here, in the title you use `unidist` from small letter, but in using_unidist/index.rts in the title `Using Unidist` with capital letter. Possible, we should align usage of capital letter in the titles and small in other cases (applicable for all files).

Reply:
aligned